src/Tools/AI Test Toolkit/app.json

```diff
@@ -0,0 +1,47 @@
+{
+    "id": "2156302a-872f-4568-be0b-60968696f0d5",
+    "name": "AI Test Toolkit",
+    "publisher": "Microsoft",
+    "version": "25.0.0.0",
+    "propagateDependencies": true,
+    "brief": "Provides tools for AI feature regression testing.",
+    "description": "The AI Test Toolkit lets developers write and run automated tests for AI features. The toolkit supports running data driven tests and the ability to get the output generated from the tests.",
+    "privacyStatement": "https://go.microsoft.com/fwlink/?LinkId=724009",
+    "EULA": "https://go.microsoft.com/fwlink/?linkid=2009120",
+    "help": "https://go.microsoft.com/fwlink/?LinkId=724011",
```

Comment:
These needs to be updated

Reply:
They are pointing to the placeholders. We can take it after merging